File: Solved problems python/ecologicalPremium.py
import sys

lines = [line.rstrip('\n') for line in open('testcasesEcologicalPremium.txt')]
# lines = [line.rstrip('\n') for line in sys.stdin]
lineCount = 0
allInput = []

for line in lines:
    allInput.append(line.split(' '))

testcases = []
totalMoney = 0
firstIndex = int(allInput[1][0]) + 2

for k in range(2, len(allInput)):
    if len(allInput[k]) != 1:
        yardSize = int(allInput[k][0])
        animals = int(allInput[k][1])
        equipmentStatus = int(allInput[k][2])
        totalMoney += round((yardSize / animals) * equipmentStatus * animals)

    if k == firstIndex or k == len(allInput) - 1:
        testcases.append(totalMoney)
        totalMoney = 0
        firstIndex += (int(allInput[k][0]) + 1)

for testcase in testcases:
    print(testcase)

File: Solved problem java/EcologicalPremium.java
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Scanner;


public class EcologicalPremium {
    static Scanner scan;
    static int totalMoney;



    public static void main(String[] args){
        scan = new Scanner(System.in);
        int i = 0, testCases = Integer.parseInt(scan.nextLine());
        while(i != testCases){
            budgetCalc();
            i++;
        }

    }

    static void budgetCalc(){
        int[] farmInformation;
        totalMoney = 0;
        int yardSize, animals, equipmentStatus;

        int rows = Integer.parseInt(scan.nextLine());
        for (int i = 0; i < rows; i++) {
            String[] line = scan.nextLine().split(" ");
            farmInformation = Arrays.stream(line).mapToInt(Integer::parseInt).toArray();
            yardSize = farmInformation[0];
            animals = farmInformation[1];
            equipmentStatus = farmInformation[2];
            totalMoney += Math.round(((float) yardSize / animals * equipmentStatus * animals));

        }

        System.out.println(totalMoney);
    }


}
